whill_modelc_publisher:
        ros__parameters:
                serialport: "/dev/ttyUSB0"
                wheel_radius: 0.135
                send_interval: 10
                publish_odom_tf: true

whill_modelc_controller:
        ros__parameters:
                serialport: "/dev/ttyUSB0"
